library.js

```diff
@@ -0,0 +1,128 @@
+const myLibrary = [];
+
+
+// Book Class
+function Book(title, author, pages, status = 'unread') {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+
+const newBook = new Book('Test Book', 'Othmane', '100');
+const secondBook = new Book('Second Book', 'Sumaer', '100');
+
+// Add book to library Function
+function addBookToLibrary(book) {
+  myLibrary.push(book);
+}
+
+addBookToLibrary(newBook);
+addBookToLibrary(secondBook);
+
+// Render Table method
+function render(table, library) {
+  for (let i = 0; i < library.length; i += 1) {
+    const book = library[i];
+    const row = table.insertRow();
+    for (let i = 0; i < Object.keys(book).length; i += 1) {
+      const key = Object.keys(book)[i];
+      const cell = row.insertCell();
+      if (key === 'status') {
+        const button = document.createElement('BUTTON');
+        button.innerHTML = book[key];
+        button.classList.add('statusButton');
+        cell.appendChild(button);
+        const deleteCell = document.createElement('BUTTON');
+        deleteCell.innerHTML = 'Delete';
+        deleteCell.classList.add('delete');
+        cell.appendChild(deleteCell);
+      } else {
+        const text = document.createTextNode(book[key]);
+        cell.appendChild(text);
```

Comment:
we have some repeating code here in `addToTable ` lets extract that out or w can just call `render` when we add to the library array and not need the addToTable.
[OPTIONAL]